### ISSUE TITLE
Fehler bei Installation auf Contao 4.8.4

### ISSUE THREAD

**Issue by johannes-bittner** [1up-lab/contao-points-of-interest#4]
Bei der Installation auf einem frischen Contao 4.8.4 erhalte ich im CM die Fehlermeldung: 
"Runtime Exeption: The plugin class "Oneup\Bundle\ContaoPointsOfInterestBundle\ContaoManager\Plugin" was not found."

Auf der Konsole: 

> λ composer require oneup/contao-points-of-interest
> Using version ^2.0 for oneup/contao-points-of-interest
> ./composer.json has been updated
> Loading composer repositories with package information
> Updating dependencies (including require-dev)
> Package operations: 1 install, 0 updates, 0 removals
>   - Installing oneup/contao-points-of-interest (2.0.1): Loading from cache
> Writing lock file
> Generating autoload files
> > ComponentInstaller\Installer::postAutoloadDump
> Compiling component files
> ocramius/package-versions:  Generating version class...
> ocramius/package-versions: ...done generating version class
> contao/manager-plugin: Generating plugin class...
> Installation failed, reverting ./composer.json to its original content.
> [RuntimeException]
> The plugin class "Oneup\Bundle\ContaoPointsOfInterestBundle\ContaoManager\Plugin" was not found.

**Comment by bytehead** [comment]
Thanks for testing, never used it with the contao manager so far 😁 
I'll provide a fix asap.

**Comment by johannes-bittner** [comment]
Very cool!

**Comment by bytehead** [comment]
Fehler ist in Version `2.0.3` behoben.

**Comment by kikmedia** [comment]
Sorry, but that doesn't solve it. I've got:

```
> php7.4 composer.phar require oneup/contao-points-of-interest
> Using version ^2.1 for oneup/contao-points-of-interest
> ./composer.json has been updated
> Running composer update oneup/contao-points-of-interest
> Loading composer repositories with package information
> Updating dependencies
> Lock file operations: 1 install, 0 updates, 0 removals
>   - Locking oneup/contao-points-of-interest (2.1.0)
> Writing lock file
> Installing dependencies from lock file (including require-dev)
> Package operations: 1 install, 0 updates, 0 removals
>   - Downloading oneup/contao-points-of-interest (2.1.0)
>   - Installing oneup/contao-points-of-interest (2.1.0): Extracting archive
> Package doctrine/reflection is abandoned, you should avoid using it. Use roave/better-reflection instead.
> Package phpunit/php-token-stream is abandoned, you should avoid using it. No replacement was suggested.
> Generating autoload files
> composer/package-versions-deprecated: Generating version class...
> composer/package-versions-deprecated: ...done generating version class
> 90 packages you are using are looking for funding.
> Use the `composer fund` command to find out more!
> contao/manager-plugin: Generating plugin class...
> 
> Installation failed, reverting ./composer.json and ./composer.lock to their original content.
> 
>                                                                                
>   [RuntimeException]                                                           
>   The plugin class "Oneup\Bundle\ContaoPointsOfInterestBundle\ContaoManager\P  
>   lugin" was not found.      
```

                                 `

**Comment by bytehead** [comment]
🙈

**Comment by bytehead** [comment]
Fixed in `2.1.1`

**Comment by johannes-bittner** [comment]
Contao 4.9.9 (cyon), module folders (/web/system/modules/points-of-interest/...) are not created, therefore the .js is missing. Call for the .css in page header is missing too ... boxes are checked: 
![grafik](https://user-images.githubusercontent.com/1597697/99638699-e4d25900-2a46-11eb-9c42-33cd3bb09362.png)
[and a very, very minor typo ...]

**Comment by bytehead** [comment]
@johannes-bittner can you open a new issue regarding this? thanks!

**Comment by bytehead** [comment]
See #6, fixed in `2.1.2`.
